frontend-html/src/plugins/tools/PluginRegistration.ts

```diff
@@ -22,9 +22,11 @@ import { registerPlugin } from "plugins/tools/PluginLibrary";
 import { AuditPlugin } from "@origam/plugin-audit";
 import { FilterPlugin  } from "@origam/plugin-filter";
 import { RadarChartPlugin  } from "@origam/plugin-chart";
+import { PivotTablePlugin  } from "@origam/plugin-pivot-table";
```

Comment:
Was this supposed to be included in this PR?

Reply:
no